packages/codemods/README.md

```diff
@@ -0,0 +1,83 @@
+# ts-morphin-migration
+
+ts-morphin-migration is a package created and managed with the `create-just` utility.
+
+## Next Steps
+
+Run
+
+```sh
+yarn
+yarn start-test
+```
+
+To start testing the codemods
+
+## To add a codemod
+
+Add your codemods to the `./src/mods` folder with `.mod.ts|tsx` as the file type.
+
+## Test with the actual package:
+
+Run
+
+```sh
+yarn build
+```
+
+To build the upgrade.js file
+
+Run
+
+```sh
+npm pack
+```
+
+from the codemods root to create a tar file for testing. Move the created tar file to the package you want to test and run
+
+```sh
+npx <tarFileName>
+```
+
+## Executing a codemod (Prototype)
+
+Make sure that fluent is installed in your package.
```

Comment:
You should list the specific package(s) here. I assume it works with fui/react but also add OUFR if it works with that.
```suggestion
Make sure that `@fluentui/react` is installed as a dependency of your package.
```

Reply:
Right now the codemods don't have a concept of package or root. That's probably a good thing to add as a filter or next steps.